packages/blocks/src/api/factory.js

```diff
@@ -260,7 +260,7 @@ const getBlockTypesForPossibleFromTransforms = ( blocks ) => {
 	const blockTypesWithPossibleFromTransforms = filter(
 		allBlockTypes,
 		( blockType ) => {
-			const fromTransforms = getBlockTransforms( 'from', blockType.name );
+			const fromTransforms = getBlockTransforms( 'from', blockType?.name );
```

Comment:
We can drop this change `getBlockTypes();` should return an array, while `getBlockType` of an unregistered type may return undefined:
<img width="550" alt="CleanShot 2021-09-16 at 15 27 51@2x" src="https://user-images.githubusercontent.com/1270189/133693747-9275755f-f0f5-4656-9cb3-62d960b5b8ee.png">

Reply:
Reverting to previous state